@xen-orchestra/lite/src/components/Select.tsx

```diff
@@ -61,9 +62,27 @@ const Select = withState<State, Props, Effects, Computed, ParentState, ParentEff
         }),
     },
   },
-  ({ additionalProps, displayEmpty = true, effects, multiple, options, required, resetState, state, ...props }) => (
+  ({
+    additionalProps,
+    displayEmpty = true,
+    effects,
+    isControlled = true,
+    multiple,
+    options,
+    required,
+    resetState,
+    state,
+    value,
```

Comment:
We can initialize the value here `value = multiple ? [] : ''`, or 
https://github.com/vatesfr/xen-orchestra/blob/899be1241826dd762cce9c5d50ae25133d9da72a/packages/xo-web/src/common/editable/index.js#L385